### PR TITLE
[CWS] ptracer: add debug logs

### PR DESCRIPTION
**Pull request by YoannGh**
### What does this PR do?

This PR adds a new `debug` logging mode to the CWS tracer:
- The previous `Debugf` function is renamed to `Logf` to match the verbose mode.
- The new `Debugf` function checks whether the debug mode is used instead.

The PR also flattens the syscall handlers in order to have a 1:1 mapping between a handler and its corresponding syscall.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/ptracer/cws.go [hunk 1]
```diff
@@ -73,7 +74,7 @@ type syscallID struct {
 }
 
 type syscallHandler struct {
-	IDs        []syscallID        // IDs defines the list of syscall IDs related to this handler
+	ID         syscallID          // ID identifies the syscall related to this handler
```
Comment (spikat) [inline pkg/security/ptracer/cws.go:77]: What's the point of changing the data structure here? It implies more code repetitions and I don't get the benefits

Comment (YoannGh) [reply pkg/security/ptracer/cws.go:77]: The benefit is that we can directly access the syscall ID from the corresponding handler like so:
https://github.com/DataDog/datadog-agent/pull/27795/files#diff-bc302f5cd1ee2f6951b5cdbee920ef217a5bb353c878dca6fb3748f8d4ad36baR487

Comment (spikat) [reply pkg/security/ptracer/cws.go:77]: I see! Yep, it could be convenient for debug. Thanks!

**Review by spikat — APPROVED** [review]

**Pull request merged**